rcl_yaml_param_parser/test/test_parser.cpp

```diff
@@ -362,6 +365,37 @@ TEST(RclYamlParamParser, test_parse_file_with_bad_allocator) {
   }
 }
 
+TEST(RclYamlParamParser, test_parse_yaml_initialize_mock) {
+  char cur_dir[1024];
+  rcutils_reset_error();
+  EXPECT_TRUE(rcutils_get_cwd(cur_dir, 1024)) << rcutils_get_error_string().str;
+
+  rcutils_allocator_t allocator = rcutils_get_default_allocator();
+  char * test_path = rcutils_join_path(cur_dir, "test", allocator);
+
+  char * path = rcutils_join_path(test_path, "correct_config.yaml", allocator);
+  ASSERT_TRUE(NULL != path) << rcutils_get_error_string().str;
+  OSRF_TESTING_TOOLS_CPP_SCOPE_EXIT(
+  {
+    allocator.deallocate(path, allocator.state);
```

Comment:
`test_path` should be deallocated too, right?

Reply:
Yes that's correct. Added.